core/utility/ast-utils.ts

```diff
@@ -40,18 +40,23 @@ function _buildRoute(options: BuildRouteOptions): string {
 
   const loadChildren = `() => import('${relativeRouteModulePath}').then((module) => module.${routeModule}Module)`;
 
-  let route = `{
-    path: '${routePath}',
-    loadChildren: ${loadChildren}
-  }`;
+  let route = (options.isChildren)
+    ? (options.isFirstRoute)
+      ? `    {\n        path: '${routePath}',\n        loadChildren: ${loadChildren}\n      }\n    `
+      : `    {\n        path: '${routePath}',\n        loadChildren: ${loadChildren}\n      }`
+    : `{\n    path: '${routePath}',\n    loadChildren: ${loadChildren}\n  }`;
 
-  if (options.isFirstRoute) {
+  if (options.isFirstRoute && !options.isChildren) {
     route = `\n  ${route}\n`;
   }
 
   return route;
 }
 
+function _removeStartComma(_str: string): string {
+  return _str.slice(1);
```

Comment:
@umbarutiba if you pass string `test string` without comma at the start, this method will remove the `t` char. Need to check if the first char is comma.